.github/workflows/tests.yml

```
@@ -128,13 +130,12 @@ jobs:
           coverage combine
           coverage xml -o reports/coverage.xml -i
 
-      - name: Check coverage
-        uses: codecov/codecov-action@v5
+      - name: Coverage comment
+        uses: MishaKav/pytest-coverage-comment@main
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          files: ./reports/coverage.xml
-          fail_ci_if_error: true
-          plugins: noop
+          pytest-xml-coverage-path: ./reports/coverage.xml
+          default-branch: develop
+          xml-skip-covered: true
```

```suggestion
          xml-skip-covered: true
          report-only-changed-files: true
```
That option was quite useful